.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "python/deps/untypy"]
+	path = python/deps/untypy
```

Comment:
place directly in python/src?

Reply:
Then it couldn't be named `untypy`. The symlink points to the untypy folder inside of the untypy repository (python modul weirdness).

Reply:
Yes, we would need some sys.path manipulation. I fear that the symlink won't work once the software is bundled und installed as a VSCode plugin (at least under Windows).